Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: up
+up:
+	RUST_LOG=hyper=error,actiserve=debug cargo run -- --private-key resources/test-key.pem
+
+.PHONY: test-all
+test-all:
+	@echo "Make sure to run 'cargo run' first"
```

Comment:
test-all could depend on run

```
test-all: run
```

and run could do exactly this

```
.PHONY: run
run:
        cargo run
```

Reply:
We need to set the path for the private key when calling `cargo run` (I don't think we want to default the private key to a test one if none is provided). Wouldn't that also mean that `run` would need to background the server process? Not the end of the world but I like being able to start and stop the server in a terminal by simply ^C ing it rather than having to pkill the process :slightly_smiling_face:

Reply:
yes, i think it would need to. and that would make it harder to stop it.  shame but i can't think of anything better.